phpBB/phpbb/language/language.php

```diff
@@ -292,10 +292,9 @@ public function lang_raw($key)
 	 *
 	 * @return string
 	 */
-	public function lang_array($key, $args = [])
+	public function lang_array($key, array $args = [])
```

Comment:
`public function lang_array($key, array $args = [])`
what if it is
`public function lang_array($key, array $args)`
and this PR moved to the 3.3.x branch?

Reply:
Elaborate?

Reply:
Not so much to elaborate I think,  If we're not in a hurry, since this seems like an extreme case, I could work it out better though. I just need the time to reproduce this mistake and work on it, which I do not think I can at the moment... sorry. The seamless solution IMHO is using `empty()` ... as mentioned above and at first glance (coding reading) - it seems to me to be the optimal solution at the moment, we're not going to bandage our heads before breaking it hopefully.

Reply:
Well, considering the ticket comments, that would be pointless. As long as we cast `$args` to `array` there're no cases we can't `count()` as it is now. So no need to replace it with `empty()` as it won't affect the result.

Reply:
Whatever you want no probs. I just expressed my opinion and I'm happy to learn. :smiley:

Reply:
IMHO that's an issue with 3.2.x and extension(s) and PHP greater than 7.2.x. 

 I do agree though changing that code... but just for 3.3.x or greater.
In this case I'd go the way I mentioned above.

> `public function lang_array($key, array $args)`
> and this PR rebased to 3.3.x

Reply:
Let's just leave this "fix" as is. it's too paltry to complicate things in such a way :)